src/main/java/com/example/petbuddybackend/service/user/UserService.java

```diff
@@ -90,29 +87,39 @@ public ProfileData getProfileData(String email) {
     @Transactional
     public PhotoLinkDTO uploadProfilePicture(String username, MultipartFile profilePicture) {
```

Comment:
przenioslbym to do PhotoService, z drugiej strony straci swoja niezaleznosc, bo bedzie musial miec UserService zeby pobrac usera.

Reply:
Właśnie z tego powodu bym nie dodawał tego tam

Reply:
ma sens